dex/networks/eth/tokens.go

```diff
@@ -199,18 +215,20 @@ var Tokens = map[uint32]*Token{
 							// map.
 							// Average of 5 transfers: 51820
 							//   [65500 48400 48400 48400 48400]
-							Swap:      175_000,
-							SwapAdd:   115_000,
-							Redeem:    87_000,
-							RedeemAdd: 33_000,
-							Refund:    67_000,
-							Approve:   65_000,
-							Transfer:  70_000,
+							//
+							// Then buffered by about 130%...
+							Swap:      242_000, // actual ~187,880 -- https://goerli.etherscan.io/tx/0x352baccafa96bb09d5c118f8dcce26e34267beb8bcda9c026f8d5353abea50fd, verified on mainnet at 188,013 gas
```

Comment:
I wonder why this is higher than either I or @JoeGruffins saw before. We both got 170k.

Reply:
Wait. Are you just using the mainnet values on testnet now?

Reply:
No, I did it on both networks.  The comment just means I verified it's the same(ish) on mainnet but without pasting my txns

Reply:
> I wonder why this is higher than either I or @JoeGruffins saw before. We both got 170k.

Same guess as in https://github.com/decred/dcrdex/pull/2013#discussion_r1100638399 with the updated contract.

Reply:
>   First redeem used 79024 gas  

Here's where we need to consider if the receiving account has ever received that token.  What I found is that it's about ~83,850 for a redeem by an account that has no balance for that token yet, but ~79,012 for any redeem after that.

There's a similar effect with transfer.  ~65,524 for an initial receive, subsequent 48,424

> Average of 5 refunds: 64299 

Not sure how it's using more gas in the estimate utility vs real life: https://goerli.etherscan.io/tx/0xc5692ad0e6d86b721af75ff3b4b7c2e17d939918db030ebf5444ccf840c7a90b

I keep observing ~59,152 when actually swapping.  Is it possible that gas used here is different if the tx sender is the same vs. different than the token recipient of the refund tx?